_how-it-works/laws-and-regulations/state-legal-fiscal-info.md

```diff
@@ -1,333 +0,0 @@
----
-title: State Legal and Fiscal Information | How It Works
```

Comment:
It might be worth it use this as an opportunity to add the `jekyll-redirect-to` plugin to redirect an empty page to the `/state-laws-and-regulations/` page